README.md

```diff
@@ -1 +1,41 @@
-# js-capstone
\ No newline at end of file
+# mobile-portfolio
+This  project is aimed at building a beautiful porfolio page.
```

Comment:
- [ ] This readme file contains inappropriate or not clear project description hence this will make your project to look unprofessional